draft-ietf-rats-corim.md

```diff
@@ -1733,19 +1741,35 @@ An ARS is a list of ECTs that describe ACS entries that are selected for use as
 {::include cddl/intrep-ars.cddl}
 ~~~
 
+## Session setup (Phase 0) {#sec-phase0}
+
+The exchange of a request for attestation appraisal for a response of Attestation Results corresponds to a single Attestation Session.
+
+During this setup phase, the Verifier populates its Appraisal Session with a consistent view of all its inputs to the Appraisal Procedure.
+Inputs are various conceptual messages collected from Reference Value Providers, Endorsers, Verifier Owners, and Attesters.
+Conceptual messages may include Attestation Evidence, CoMID tags {{sec-comid}}, CoSWID tags {{-coswid}}, CoBOM tags {{sec-cobom}}, and static cryptographic validation key material (including raw public keys, root certificates, intermediate CA certificate chains, and Concise Trust Anchor Stores (CoTS) {{-ta-store}}.
```

Comment:
Yeah.. Ned's change to add parentheses didn't treat this one line consistently, and adding parens only for CoBOM seems weird, no?

Reply:
> [...] adding parens only for CoBOM seems weird, no?

It makes sense to add parentheses because external refs are rendered differently from internal refs:

* `{{EXT}}` becomes `[EXT]`
* `{{INT}}` becomes `INT`

Reply:
Added to both cobom and comid references then.